Sources/Sentry/SentrySpan.m

```diff
@@ -130,15 +131,15 @@ - (NSDictionary *)serialize
 
     @synchronized(_data) {
         if (_data.count > 0) {
-            mutableDictionary[@"data"] = _data.copy;
+            mutableDictionary[@"data"] = [_data.copy sentry_sanitize];
         }
     }
 
     @synchronized(_tags) {
         if (_tags.count > 0) {
             NSMutableDictionary *tags = _context.tags.mutableCopy;
             [tags addEntriesFromDictionary:_tags.copy];
-            mutableDictionary[@"tags"] = tags;
+            mutableDictionary[@"tags"] = [tags sentry_sanitize];
```

Comment:
Since it's impossible to set a tag other than a simple string, we probably don't need to call sentry_sanitize here, correct? I can remove it. It's also not possible to test this since I can't set a Date as a tag for example.

Reply:
Yep, please remove it.